### PR TITLE
A heartbeat url

### PR DESCRIPTION
**Pull request by jbzdak**

### CODE REVIEW

File: peerinst/views.py [hunk 1]
```diff
@@ -470,6 +480,81 @@ def get_context_data(self, **kwargs):
...
+            if response_content != canary:
+                return HeartbeatUrlCheckResult(False, "Cant load uploaded file")
+            return HeartbeatUrlCheckResult(True, "Media upload works")
+        except Exception:
+            return HeartbeatUrlCheckResult(False, "Some error in media upload")
+        finally:
+            default_storage.delete(path)
+
+    def get(self, request):
+
```
Comment (antoviaque) [inline peerinst/views.py]: Extra whitespace

File: peerinst/views.py [hunk 2]
```diff
@@ -470,6 +480,81 @@ def get_context_data(self, **kwargs):
...
+        checks_ok = all((check.is_ok for check in checks))
+
+        status = 200 if checks_ok else 500
+
+        return TemplateResponse(
+            request, "peerinst/heartbeat.html", context={"checks": checks},
+            status=status)
+
+
```
Comment (antoviaque) [inline peerinst/views.py]: It's missing tests.

**Comment by antoviaque** [comment]
:+1: once the comments are addressed & the tests pass. If needed for the merge today, the changes can be done in a separate PR (but still part of the same task).

**Comment by jbzdak** [comment]
I'll add the tests as a follow-up PR.

**Pull request merged**